### ISSUE TITLE
currentHandle not initialized?

### ISSUE THREAD

**Issue by sabiland** [joncardasis/ChromaColorPicker#55]
I have noticed that currentHandle is not initialized until you move one handle first? This means, that if you first try to change brightness slider, handle (color) is not affected.

I've fixed this with:

```
// The last active handle.
// SABI TWEAK
// SABI TWEAK
// SABI TWEAK
public var currentHandle: ChromaColorHandle?
//private(set) public var currentHandle: ChromaColorHandle?
```

And setting currentHandle manually.

**Comment by bennnjamin** [comment]
Same issue here. Can you share more code examples of how you set the handle manually?

**Comment by sabiland** [comment]
After you modify ChromaColorPicker source file (in my first comment), you can set current handle anywhere where you need it.

```
let firstHandle = ChromaColorHandle()
firstHandle.color = settedColor
fullColorPicker.addHandle(firstHandle)

// NOTE: SABI TWEAK
// NOTE: SABI TWEAK
// NOTE: SABI TWEAK
fullColorPicker.currentHandle = firstHandle
```

**Comment by bennnjamin** [comment]
@sabiland looks like this was actually fixed by pull request #49 but for some reason it's not pulled in by CocoaPods. I fixed the issue with no code changes by updating my ```Podfile```

```
pod 'ChromaColorPicker', '~> 2.0.2'
```

or

```
pod 'ChromaColorPicker', :git => 'https://github.com/joncardasis/ChromaColorPicker', :branch => 'master'
```

**Comment by sabiland** [comment]
Great thx for info, it works by updating pod.
